[UPD] parse more fields as of

hbrunn opened the pull request:
https://www.ing.nl/media/ING_ming_mt940s_24_juli_tcm162-46356.pdf
p 8ff

hbrunn committed:
[UPD] parse more fields for MT940

as of https://www.ing.nl/media/ING_ming_mt940s_24_juli_tcm162-46356.pdf
p 8ff

NL66278 commented:
:+1: LGTM In general. But I have seen while testing that putting EREF also in the message field will lead to duplicate info.

NL66278 commented on account_bank_statement_import_mt940_base/mt940.py:
```diff
@@ -83,14 +83,28 @@ def handle_common_subfields(transaction, subfields):
+            else
+            '/'.join(x for x in subfields['REMI'] if x)
+        )
+    # EREF: End-to-end reference
```
I think EREF should not be put into message (which will get into name), because in many cases it will lead to duplicate information. Eref will already end up in the ref field of the bank statement line.

NL66278 committed:
[FIX] There is no CR or LF in mt940 data to be parsed.

The calling logic from the MT940 parser concatenates all lines from a mt940 tag.

hbrunn committed:
Merge pull request #1 from NL66278/8.0-mt940-ing-update-2016-09-29

[FIX] There is no CR or LF in mt940 data to be parsed.
